pkg/db/interface.go

```diff
@@ -105,11 +128,12 @@ func NewWithSQLConn(db *sql.DB) VizierDBInterface {
 }
 
 func New() VizierDBInterface {
-	db, err := sql.Open(dbDriver, getDbName())
-	if err != nil {
+	if db, err := openSQLConn(dbDriver, getDbName(), connectInterval, connectTimeout); err != nil {
 		log.Fatalf("DB open failed: %v", err)
+		return nil
+	} else {
```

Comment:
AFAIK, go style avoids this extra else{} block.

Reply:
Just avoided declaration of db outside if statement. I'm not sure which is better,

Reply:
How about retaining the old code style and just replacing the sql.Open call with openSQLConn? :-)